src/platform-integration/web/wasm.md

```diff
@@ -116,15 +121,11 @@ If the application doesn't load:
 
 Wasm support has some limitations. The following list covers the common issues.
 
-#### Chrome-only, with flags
-
-As mentioned [earlier](#load-it-an-a-browser), to compile web apps to Wasm,
-install Chrome 112 or later with experimental flags enabled.
+#### Chrome or Firefox nightly, with flags
 
-At this time, the Dart Wasm compiler leverages the
-[proposed JavaScript-Promise Integration (JSPI) feature](https://github.com/WebAssembly/js-promise-integration/blob/main/proposals/js-promise-integration/Overview.md).
-Firefox doesn't support the JSPI proposal. Once Dart migrates away from JSPI,
-Firefox should work with the appropriate flags.
+As mentioned [earlier](#load-it-an-a-browser), to run Flutter web apps compiled
+to Wasm, use _Chrome 113 or later_ or a _Firefox nightly build_ with experimental flags
+enabled.
```

Comment:
Let's be more explicit with the link since readers may not be reading it in order.

```suggestion
As mentioned in [Load it in a browser](#load-it-an-a-browser), 
to run Flutter web apps compiled to Wasm, 
use _Chrome 113 or later_ or a _Firefox nightly build_ with 
experimental flags enabled.
```